source/simulator/core.cc

```diff
@@ -344,8 +344,11 @@ namespace aspect
         AssertThrow ( parameters.pressure_normalization == "no",
                       ExcMessage("The free surface scheme can only be used with no pressure normalization") );
 
-        // Allocate the FreeSurfaceHandler object
-        free_surface = std_cxx14::make_unique<FreeSurfaceHandler<dim>>(*this, prm );
+        // Allocate the MeshDeformationHandler object
+        //free_surface = std_cxx14::make_unique<MeshDeformation::MeshDeformationHandler<dim>>(*this);
+        free_surface.reset(new MeshDeformation::MeshDeformationHandler<dim>(*this));
```

Comment:
I think the previous (commented out) line is the canonical syntax we are using these days